subsys/net/ip/net_if.c

```diff
@@ -367,11 +367,29 @@ struct net_if *net_if_lookup_by_dev(struct device *dev)
 
 struct net_if *net_if_get_default(void)
 {
+	struct net_if *iface = NULL;
+
 	if (__net_if_start == __net_if_end) {
-		return NULL;
+		return iface;
```

Comment:
I wouldn't make this change. The original code makes (more) clear that the return value is NULL.

Reply:
I agree with Paul